### PR TITLE
Add alternative to jQuery toggleClass.

### PR DESCRIPTION
**Pull request by damieng**

### CODE REVIEW

File: comparisons/elements/toggle_class/ie10.js [hunk 1]
```diff
@@ -0,0 +1,4 @@
+if (el.classList.contains(className))
+    el.classList.remove(className);
```
Comment (zackbloom) [inline comparisons/elements/toggle_class/ie10.js]: There's actually a `classList.toggle` method: https://developer.mozilla.org/en-US/docs/Web/API/Element.classList

**Comment by zackbloom** [comment]
Looks great, switch to the `classList` toggle and I'll merge.

**Comment by damieng** [comment]
Switched to classList.toggle.

File: comparisons/elements/toggle_class/ie10.js [hunk 2]
```diff
@@ -0,0 +1 @@
+el.classList.toggle(className);
```
Comment (zackbloom) [inline comparisons/elements/toggle_class/ie10.js]: Something strange happened here, it might be a bug with github.

Comment (damieng) [inline comparisons/elements/toggle_class/ie10.js]: I see some &lt;span...&gt; after that when in the "Files Changed" tab in GitHub. It certainly isn't in the underlying raw file.

Comment (damieng) [inline comparisons/elements/toggle_class/ie10.js]: Added blank lines to the files to make them consistent with the other files and avoid the GitHub bug.

**Comment by zackbloom** [comment]
Thanks @damieng

**Pull request merged**